NEWS

```diff
@@ -2,6 +2,34 @@
 
 Sandia OpenSHMEM NEWS -- history of user-visible changes.
 
+v1.4.4
+------
+- Experimental support for RXM/Verbs provider stack with libfabric 1.8.
+  Requires --enable-hard-polling and --enable-ofi-mr=basic build flags.  The
+  FI_MR_CACHE_MAX_COUNT=0 environment variable should be set when threading is
```

Comment:
Should we re-write "`FI_MR_CACHE_MAX_COUNT` environment variable should be set to 0"?